pages/common/vcsh.md

```diff
@@ -0,0 +1,27 @@
+# vcsh
+
+> Version Control System for `$HOME` - multiple Git repositories in `$HOME`.
```

Comment:
I am not exactly clear what this does. The `-` in the middle is a bit odd. Can you explain a bit ?

Reply:
I changed the description from the official title to a more descriptive one.